src/opaque.c

```diff
@@ -1278,7 +1278,7 @@ int opaque_RecoverCredentials(const uint8_t _resp[OPAQUE_SERVER_SESSION_LEN/*+en
 }
 
 // extra function to implement the hmac based auth as defined in the ietf cfrg draft
-int opaque_UserAuth(uint8_t _ctx[OPAQUE_SERVER_AUTH_CTX_LEN], const uint8_t authU[crypto_auth_hmacsha256_BYTES], const Opaque_App_Infos *infos) {
+int opaque_UserAuth(const uint8_t _ctx[OPAQUE_SERVER_AUTH_CTX_LEN], const uint8_t auth[crypto_auth_hmacsha256_BYTES], const Opaque_App_Infos *infos) {
```

Comment:
dunno, since there are two auth tokens, one from the server and one from the user it might make sense to be more explicit about which auth token we are referring to also in the var name.

Reply:
I will remove this part from this commit.

In a separate commit, I plan on renaming `auth` in `opaque_RecoverCredentials` to `authU` since `opaque_RecoverCredentials(auth)/opaque_UserAuth(authU)` are paired.

Is the auth token from the server the `ctx` here?

Reply:
thx

Reply:
I renamed `auth` to `authU` in https://github.com/stef/libopaque/pull/6/commits/8be9cceccb0082751638b633413835f099c44ec9 and `ctx` to `sec` in https://github.com/stef/libopaque/pull/6/commits/bed5310b4eeb7e4610b60c2ea9d7f449d9689991.